### PR TITLE
RSP Zeropoint Tutorial notebook

### PR DESCRIPTION
**Pull request by P-1884**
A notebook on retrieving and verifying the magnitude zeropoints from random single-exposures/coadds from the RSP.

### CODE REVIEW

**Comment by nkhadka21** [comment]
Hi @P-1884, It would be great if you could fix Pre-commit error.

File: notebooks/RSP_Zeropoint_Tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,289 @@
+{
```
Comment (sibirrer) [inline notebooks/RSP_Zeropoint_Tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #12.    from lsst_science_pipeline import tap_query, list_of_calexp, warp_to_exposure
</pre><p>can you make this line assuming you have installed SLSim and you can state: from slsim.lsst_science_pipeline import ...?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Review by sibirrer — COMMENTED** [review]
Thank you very much @P-1884 ! Very useful!
I had one minor comment and leave it to @nkhadka21 as he works with the RSP.

**Comment by nkhadka21** [comment]
Hi @P-1884 , I think this tutorial is more like a validation of dp0 data. Therefore, I think you can create a another folder with name "dp0_validation_notebook" or similar name and put this notebook there. The current notebook folder is for slsim user guide. Thank you!

**Review by nkhadka21 — APPROVED** [review]
@P-1884 , thank you you very much for this tutorial! I merged it.

**Pull request merged**